eclipse-plugin/service/cc.codechecker.service.api/src/main/java/cc/codechecker/api/runtime/CodecheckerServerThread.java

```diff
@@ -11,7 +11,8 @@
 
 public class CodecheckerServerThread {
 
-    public final int serverPort = (new Random()).nextInt(10000) + 15000;
+    public final int serverPort = (new Random()).nextInt(10000) + 5000 + Math.min(
```

Comment:
Could you write some comments here why the random server port this calculated this way?